andela/eventx
# #[109795056 ] create and design bookings page for users and  event mangers.

#### What does this PR do?
allow users/ managers have access to view each bookings of all events created/attending
allow users/mangers have access to print each ticket type of a booking in pdf format
allow users/mangers have access to download each ticket type of a booking in pdf format

#### Description of Task to be completed?
create a route for bookings view
create a route for each booking view
create a print path for each ticket type
create a route for download of all tickets

#### How should this be manually tested?
- clone this repository  and test from your locallhost 

for **new users** after sign up
- fill in the form to become an event manager
- create an event and ensure tickets are booked for the event

for **exisiting users**
- to view the bookings directly visit http:localhost:3000/tickets
- to view each bookings visit "my account" which is the user dashboard 
- click on "view booking" it would redirect you to a url like this `http:localhost:3000/events/1/tickets`


#### What are the relevant pivotal tracker stories?
#109795056
